tests/tests_suite_test.go

```diff
@@ -381,20 +304,101 @@ func cmdWithRetry(cmd Cmd, expectedResult string, timeout int) bool {
 		}
 		time.Sleep(1 * time.Second)
 	}
-	fmt.Printf("FAIL: '%s' does not match expected result of '%s'\n", result, expectedResult)
+	fmt.Fprintf(GinkgoWriter, "FAIL: '%s' does not match expected result of '%s'\n", result, expectedResult)
 	return false
 }
 
 // gitInit simply invokes 'git init' and verifies the command is successful
 func gitInit() {
-	cmd, err := start("git init")
+	cmd, err := start("git init", "")
 	Expect(err).NotTo(HaveOccurred())
 	Eventually(cmd).Should(Say("Initialized empty Git repository"))
 }
 
 // gitClean destroys the .git directory and verifies the command is successful
 func gitClean() {
-	cmd, err := start("rm -rf .git")
+	cmd, err := start("rm -rf .git", "")
 	Expect(err).NotTo(HaveOccurred())
 	Eventually(cmd).Should(Exit(0))
 }
+
+func registerHTTP(url string, username string, password string, email string) string {
+	registerURL := fmt.Sprintf("%s/v2/auth/register/", url)
```

Comment:
If we plan on expanding the use of making API calls as done here and in `loginHTTP` below, it would be handy to have both of these (and future methods) invoke a helper function, something like:
```
func makeHTTPReq(url, method string, jsonData []byte) string {
  req, err := http.NewRequest(method, url, bytes.NewBuffer(jsonData))
  req.Header.Set("Content-Type", "application/json")

  client := &http.Client{}
  resp, err := client.Do(req)
  if err != nil {
    panic(err)
  }
  defer resp.Body.Close()

  respBody, _ := ioutil.ReadAll(resp.Body)
  return string(respBody)
}